ui/src/consts.ts

```diff
@@ -0,0 +1,19 @@
+export enum ChangeTypeEnum {
+    Created = "create",
+    Updated = "update",
+    Deleted = "delete"
+}
+
+export const createClass = ChangeTypeEnum.Created
+export const updateClass = ChangeTypeEnum.Updated
+export const deleteClass = ChangeTypeEnum.Deleted
+export const infoClass = "info"
+
+type EnumDictionary<T extends string | symbol | number, U> = {
+    [K in T]: U;
+};
+export const TypeCaptionDictionary: EnumDictionary<string, string> = {
```

Comment:
try to use https://stackoverflow.com/questions/6215349/getting-corresponding-enum-key-for-the-value-passed or other solution
for taking the enum key by value